.travis.yml

```diff
@@ -76,8 +76,7 @@ before_script:
   - composer install --prefer-dist --no-interaction
 
 script:
-  - if [[ $DEFAULT = 1 && $TRAVIS_PHP_VERSION = 7.* ]]; then export CODECOVERAGE=1 ; phpdbg -qrr vendor/bin/phpunit --coverage-clover=clover.xml; fi
```

Comment:
We can keep a separate branch where we keep the line, so that they can debug until they fix the problem(s)